libc/CMakeLists.txt

```diff
@@ -245,6 +245,7 @@ option(LIBC_INCLUDE_DOCS "Build the libc documentation." ${LLVM_INCLUDE_DOCS})
 
 include(CMakeParseArguments)
 include(LLVMLibCCheckCpuFeatures)
+include(check_compiler_features)
```

Comment:
This name should match the case of the other cmake utilities

Reply:
Fixed.